resolvers/relay.ts

```diff
@@ -0,0 +1,66 @@
+export type RelayInformation = {
+    name?: string;
+    description?: string;
+    pubkey?: string;
+    contact?: string;
+    supported_nips?: number[];
+    software?: string;
+    version?: string;
+    icon?: string;
+};
+
+export const Information = async (kv: Deno.Kv) => {
+        const res = await kv.get<RelayInformation>(["relay_information"]);
+        if (res.value == null) return null;
+        return res.value;
+}
+
+export class RelayInformationStore {
+    not_modifiable_information: RelayInformation = {
```

Comment:
remove `RelayInformation` type. It's better to use smaller type when possible.
`RelayInformation` is bigger than necessary here.

Reply:
ok